cli-commands/compiler/etherlime-blockchain-utils/index.js

```diff
@@ -0,0 +1,65 @@
+var Blockchain = {
```

Comment:
Do we need this file and why?

Reply:
Yes, we need it. It's used in contract.js.